package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "SmartSyncExplorerReactNative",
+    "version": "5.0.0",
+    "private": true,
+    "scripts": {
+        "start": "node node_modules/react-native/local-cli/cli.js start",
+        "test": "jest"
+    },
+    "dependencies": {
+        "react": "15.3.2",
+        "react-native": "0.35.0",
+        "salesforcemobilesdk-ios": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git#unstable",
```

Comment:
Will we change these branches to `master` when we release? Should we add an `unstable` branch here as well to make branch specs easier across the board?